### PR TITLE
hotfix: blinking when expanded and collapsed

### PR DESCRIPTION
**Pull request by youjung-hong**
### Please check if the PR fulfils these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/factory/calendar.js [hunk 1]
```diff
@@ -708,8 +710,10 @@ Calendar.prototype.move = function(offset) {
 
             childView.setState({
                 collapsed: true
-            });
+            }, true);
         });
+
+        view.render();
```
Comment (youjung-hong) [inline src/js/factory/calendar.js]: `dayGrid.setState()` 내에서 parentView.render()를 호출
패널이 3개이면 불필요하게 parentView.render()가 3번 호출됨

=> view.render()로 변경함

File: src/js/handler/daygrid/click.js [hunk 2]
```diff
@@ -91,15 +91,11 @@ DayGridClick.prototype._onClick = function(clickEvent) {
...
-        this.view.setState({
-            clickedExpandBtnIndex: parseInt(domutil.getData(expandBtnElement, 'index'), 10)
-        });
-
         /**
          * click expand btn event
          * @events DayGridClick#expand
          */
-        self.fire('expand');
+        self.fire('expand', parseInt(domutil.getData(expandBtnElement, 'index'), 10));
```
Comment (youjung-hong) [inline src/js/handler/daygrid/click.js]: setState() 시 parent.render()를 타므로, view의 expand 핸들러에서 한번에 setState 하도록 함

File: src/js/view/week/dayGrid.js [hunk 3]
```diff
@@ -93,7 +93,10 @@ DayGrid.prototype.getBaseViewModel = function(viewModel) {
             }
         }
 
-        visibleScheduleCount = Math.floor(panelHeight / (opt.scheduleHeight + opt.scheduleGutter));
+        visibleScheduleCount = Math.floor(
+            (panelHeight + opt.scheduleContainerTop) / (opt.scheduleHeight + opt.scheduleGutter)
+        );
```
Comment (youjung-hong) [inline src/js/view/week/dayGrid.js]: 계산 오류 수정, panel.setHeight할 때 opt.scheduleContainerTop을 넣어주므로 이렇게 하는 게 맞음

File: src/js/view/week/dayGrid.js [hunk 4]
```diff
@@ -195,46 +199,82 @@ DayGrid.prototype._expand = function() {
...
+        viewModel.matrices = matrices;
+    }
+
+    if (this.children.length) {
+        this.children.each(function(childView) {
+            heights.push(childView.getHeight(viewModel));
+        });
+    }
+
+    return heights;
```
Comment (youjung-hong) [inline src/js/view/week/dayGrid.js]: _expand() 또는 _collapse() 시에 딱 맞는 높이를 구하도록 함

File: src/js/view/week/week.js [hunk 5]
```diff
@@ -88,6 +89,46 @@ util.inherit(Week, View);
  * @override
  */
 Week.prototype.render = function() {
+    var viewModel = this.getBaseViewModel();
```
Comment (youjung-hong) [inline src/js/view/week/week.js]: getBaseViewModel() 함수 분리

**Comment by dongsik-yoo** [comment]
Close this and will re-write it.

**Pull request closed without merging**